generators/server/templates/src/main/java/package/web/filter/SpaWebFilter.java.ejs

```diff
@@ -35,14 +35,19 @@ public class SpaWebFilter extends OncePerRequestFilter {
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
         throws ServletException, IOException {
         String path = request.getRequestURI();
-        if (!path.startsWith("/api") && !path.startsWith("/management")<% if (authenticationTypeOauth2) { %> && !path.startsWith("/login") && !path.startsWith("/oauth2")<% } %>
-<%_ if (applicationTypeGateway || applicationTypeMonolith) { _%>
-            && !path.startsWith("/services") && !path.startsWith("/v3/api-docs")
+        if (
+            !path.startsWith("/api")
+            && !path.startsWith("/management")
+            && !path.startsWith("/v3/api-docs")
```

Comment:
`/v3/api-docs` exists in microservices.

Reply:
When I create a new project, Prettier formats this like the following:

```java
@Override
protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
    throws ServletException, IOException {
    String path = request.getRequestURI();
    if (
        !path.startsWith("/api") &&
        !path.startsWith("/management") &&
        !path.startsWith("/v3/api-docs") &&
        !path.contains(".") &&
        path.matches("/(.*)")
    ) {
        request.getRequestDispatcher("/index.html").forward(request, response);
        return;
    }

    filterChain.doFilter(request, response);
}
```